docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc

```diff
@@ -0,0 +1,80 @@
+.Drain source node
+
+. On the source node, back up the `/etc/ceph/` directory to run `cephadm` and get a shell for the {Ceph} cluster from the source node:
++
+----
+$ mkdir -p $HOME/ceph_client_backup
+$ sudo cp -R /etc/ceph $HOME/ceph_client_backup
+----
+
+. Identify the active `ceph-mgr` instance:
++
+----
+$ cepahdm shell -- ceph mgr stat
+----
+
+. Fail the `ceph-mgr` if it is active on the source node or target node:
++
+----
+$ cephadm shell -- ceph mgr fail <mgr_instance>
+----
++
+* Replace `<mgr_instance>` with the Ceph Manager daemon to fail.
+
+. From the `cephadm` shell, remove the labels on the source node:
++
+----
+for label in mon mgr _admin; do
+    cephadm shell -- ceph orch host rm label <source_node> $label;
```

Comment:
The rest of the procedure is italicized, and it starts here. Its not obvious to me what's causing it.  After we eventually merge this I can take a look in VS Code.